app/views/articles/_liquid.html.erb

```diff
@@ -7,22 +7,44 @@
         <%= article.video_duration_in_minutes %></span>
     </a>
   <% end %>
-  <a href='<%= article.user.path %>' class='ltag__link__link'>
-    <div class='ltag__link__pic'>
-      <img src='<%= Images::Profile.call(article.user.profile_image_url, length: 150) %>' alt='<%= "#{article.user.username} image" %>'>
-    </div>
-  </a>
-  <a href='<%= article.path %>' class='ltag__link__link'>
-    <div class='ltag__link__content'>
-      <h2><%= title %></h2>
-      <h3><%= article.user.name %> ・ <%= article.readable_publish_date %> ・ <%= article.reading_time < 2 ? 1 : article.reading_time %> min read</h3>
-      <div class='ltag__link__taglist'>
-      <% article.tag_list.each do |t| %>
-        <span class='ltag__link__tag'>#<%= t %></span>
-      <% end %>
+  <% if article.organization %>
+    <a href='<%= article.organization.path %>' class='ltag__link__link'>
+      <div class='ltag__link__org__pic'>
+        <img src='<%= Images::Profile.call(article.organization.profile_image_url, length: 150) %>' alt='<%= "#{article.organization.name} image" %>'>
+        <div class='ltag__link__user__pic'>
+          <img src='<%= Images::Profile.call(article.user.profile_image_url, length: 150) %>' alt='<%= "#{article.user.username} image" %>'>
+        </div>
       </div>
-    </div>
-  </a>
+    </a>
+    <a href='<%= article.path %>' class='ltag__link__link'>
+      <div class='ltag__link__content'>
+        <h2><%= title %></h2>
+        <h3><%= article.user.name %> for <%= article.organization.name %> ・ <%= article.readable_publish_date %> ・ <%= article.reading_time < 2 ? 1 : article.reading_time %> min read</h3>
+        <div class='ltag__link__taglist'>
+        <% article.tag_list.each do |t| %>
+          <span class='ltag__link__tag'>#<%= t %></span>
+        <% end %>
+        </div>
+      </div>
+    </a>
+  <% else %>
+    <a href='<%= article.user.path %>' class='ltag__link__link'>
+      <div class='ltag__link__pic'>
+        <img src='<%= Images::Profile.call(article.user.profile_image_url, length: 150) %>' alt='<%= "#{article.user.username} image" %>'>
```

Comment:
nitpick (blocking) - like the example above, removing the word "image" helps reduce unnecessary noise for screen reader users. I know it was like this before, but it's a great opportunity to sweep it up now 🙌 

```suggestion
        <img src='<%= Images::Profile.call(article.user.profile_image_url, length: 150) %>' alt='<%= "#{article.user.username}" %>'>
```